cg/cli/__init__.py

```diff
@@ -1 +0,0 @@
-from .base import base
```

Comment:
If anyone knows what this does please enlighten me. It was the root of a nasty circular import, and all tests pass when removing it.